packages/dashboard/src/components/app.tsx

```diff
@@ -44,6 +44,7 @@ export default function App(): JSX.Element | null {
   const [authInitialized, setAuthInitialized] = React.useState(!!appConfig.authenticator.user);
   const [authenticator, setAuthenticator] = React.useState(appConfig.authenticator);
   const [user, setUser] = React.useState<User | null>(appConfig.authenticator.user || null);
+  const [ws, setWs] = React.useState<WebSocket | undefined>(undefined);
```

Comment:
```suggestion
  const [ws, setWs] = React.useState<WebSocket>(() => new WebSocket(appConfig.trajServerUrl));
```

Would this do the same thing as the effect?

Reply:
Tested and it seems to be fine.